### PR TITLE
fix: fix console errors

### PR DESCRIPTION
**Pull request by Doug-Fernando**

### CODE REVIEW

**Comment by lucasportella** [comment]
can you tell me which console errors this fixes and where they occur?

**Comment by Doug-Fernando** [comment]
Home page
image
image
image
image
image
image
image

**Comment by Doug-Fernando** [comment]
Blocks Page
image
image

**Comment by Doug-Fernando** [comment]
Transactions Page
image
image

**Comment by Doug-Fernando** [comment]
Assets Page
image
image

**Comment by Doug-Fernando** [comment]
Marketplaces Page
image

File: src/components/InputGlobal/index.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ const Input: React.FC<InputGlobal> = ({
     placeholder,
     onChange: handleInput,
     onKeyDown: keyDownHandle,
-    setOpenSearch: setOpenSearch,
+    // setOpenSearch: setOpenSearch,
```
Comment (samufacanha2) [inline src/components/InputGlobal/index.tsx:89]: either remove or keep the line, don't comment it

**Review by samufacanha2 — CHANGES_REQUESTED** [review]
check comment

**Review by samufacanha2 — APPROVED** [review]

**Review by lucasportella — APPROVED** [review]

**Pull request merged**